### PR TITLE
Project report

### PR DESCRIPTION
**Pull request by njkim**
Added the rest of the tabs.
- substance, temporal relations, parameters &. outcomes, parthood

### CODE REVIEW

File: afs/media/js/views/components/reports/project.js [hunk 1]
```diff
@@ -38,16 +47,81 @@ define([
...
+                                type: 'resource'
+                            }]
+                        }
+                    ]
+            });
+            self.temporalData = ko.observable({
+                sections: [
+                    {
+                        title: "Temporal Relations of Project", 
+                        data: [{
```
Comment (aarongundel) [inline afs/media/js/views/components/reports/project.js:84]: Period is an inactive model - after discussing with cyrus, let's comment this (the reference to period) out for now.

File: afs/templates/views/components/reports/project.htm [hunk 2]
```diff
@@ -58,6 +58,44 @@ <h1 class="afs-report-toolbar-title"><span class="afs-report-name">{% trans "Pro
...
+            </div>
+            <!-- Parameters & Outcomes Tab -->
+            <div class="afs-report-page" data-bind="if: activeSection() === 'parameters'">
+                <div data-bind="component: {
+                    name: 'views/components/reports/scenes/default',
+                    params: {
+                        data: parameterData,
+                    }
+                }"></div>
+            </div>
```
Comment (aarongundel) [inline afs/templates/views/components/reports/project.htm:98]: Can you add the JSON tab in here?

File: afs/media/js/views/components/reports/project.js [hunk 3]
```diff
@@ -14,6 +14,10 @@ define([
             Object.assign(self, reportUtils);
             self.sections = [
                 {'id': 'name', 'title': 'Names and Classifications'},
+                {'id': 'substance', 'title': 'Substance'},
```
Comment (aarongundel) [inline afs/media/js/views/components/reports/project.js:17]: nit:  I've been removing the single quotes from id/title - can you remove those here?

File: afs/media/js/views/components/reports/project.js [hunk 4]
```diff
@@ -38,16 +47,81 @@ define([
...
+                                card: self.cards?.['parent project'],
+                                type: 'resource'
+                            }]
+                        }
+                    ]
+            });
+            self.temporalData = ko.observable({
+                sections: [
+                    {
+                        title: "Temporal Relations of Project",
```
Comment (aarongundel) [inline afs/media/js/views/components/reports/project.js:83]: nit: I've been removing the double quotes from strings - can you remove those here and move to single quotes for strings in this file?

**Review by aarongundel — CHANGES_REQUESTED** [review]
looks good!  just a few nits

File: afs/media/js/views/components/reports/project.js (hunk 1 above)
Comment (njkim) [reply afs/media/js/views/components/reports/project.js]: Commented out the section

File: afs/templates/views/components/reports/project.htm (hunk 2 above)
Comment (njkim) [reply afs/templates/views/components/reports/project.htm:98]: added

File: afs/media/js/views/components/reports/project.js (hunk 3 above)
Comment (njkim) [reply afs/media/js/views/components/reports/project.js]: fixed

File: afs/media/js/views/components/reports/project.js (hunk 4 above)
Comment (njkim) [reply afs/media/js/views/components/reports/project.js]: fixed

**Review by aarongundel — APPROVED** [review]

**Pull request merged**